README.md

```diff
@@ -158,12 +164,12 @@ In this section, we'll use the [Azure CLI](https://docs.microsoft.com/cli/azure/
    az postgres server firewall-rule create --resource-group $rgName --server-name $serverName --name AllowAllAzureIps --start-ip-range 0.0.0.0 --end-ip-range 0.0.0.0
    ```
 
-1. Add configuration values to the Web App
+1. Add configuration values to the Web App (App Service)
 
    ```bash
-   az webapp config appsettings set --resource-group $rgName --name $webAppName --setting STORAGE_ACCOUNT=$saName
-   az webapp config appsettings set --resource-group $rgName --name $webAppName --setting STORAGE_ACCOUNT_KEY=$saKey
-   az webapp config appsettings set --resource-group $rgName --name $webAppName --setting STORAGE_ACCOUNT_CONTAINER=$container
+   az webapp config appsettings set --resource-group $rgName --name $webAppName --setting AZURE_STORAGE_ACCOUNT=$saName
```

Comment:
This env variables weren't matching the example, I've updated them so it will work when app is launched and you try and upload images.

Reply:
It's almost like I copy/pasted the previous one 😉